third_party/skia/METADATA

```diff
@@ -11,12 +11,11 @@ third_party {
     type: GIT
```

Comment:
We should probably update this to new format as per [go/cobalt-upstream-repo-tracking](https://goto.google.com/cobalt-upstream-repo-tracking). Instead of `url`, you'll have `identifier` values here.